scripts/user/setup-glued-users.ts

```diff
@@ -152,7 +150,22 @@ const getPractitionerUsers = async (): Promise<any[]> => {
       .execute(),
   );
 
-  return users;
+  const dojIrsPractitioners = await getDbReader(reader =>
+    reader
+      .selectFrom('dwUser')
+      .selectAll()
+      .where('role', '=', 'irsPractitioner')
+      .where('practiceType', '=', 'DOJ')
+      .execute(),
```

Comment:
Do we need all of them? Should we add a limit or something?

```suggestion
      .where('practiceType', '=', 'DOJ')
      .limit(10)
      .execute(),
```

Reply:
there are only 15 dojpractitioners so its not too much to add them all

Reply:
should i leave this as is for now? or do we predict this number growing substantially over the life of the application?

Reply:
Should be fine for now. We don't anticipate this number growing substantially.